optuna_dashboard/preferential/_study.py

```diff
@@ -265,7 +266,12 @@ def get_best_trials(study_id: int, storage: optuna.storages.BaseStorage) -> list
     ]
     preferences = get_preferences(study_id, storage)
     worse_numbers = {worse for _, worse in preferences}
-    return [copy.deepcopy(t) for t in ready_trials if t.number not in worse_numbers]
+    skiped_numbers = get_skiped_trials(study_id, storage)
+    return [
+        copy.deepcopy(t)
+        for t in ready_trials
+        if t.number not in worse_numbers and t.number not in skiped_numbers
+    ]
```

Comment:
I would say it should be better to avoid getting trial_id from the system_attr key like `skiped_trials.append(int(k.split(":")[-1]))`.

How about writing this logic like the following instead?

```suggestion
    study_system_attrs = storage.get_study_system_attrs(study_id)
    best_trials = []
    for t in ready_trials:
        if t.number not in worse_numbers:
            continue
        skipped_key = _SYSTEM_ATTR_PREFIX_SKIP_TRIAL + str(t._trial_id)
        if skipped_key in study_system_attrs:
            continue
        best_trials.append(t)
    return best_trials
```